unitTests/ui/QuerySummaryTest.ts

```diff
@@ -405,7 +405,7 @@ export function QuerySummaryTest() {
         test.env.queryStateModel.get = () => 'querySearched';
         Simulate.query(test.env, { results: FakeResults.createFakeResults(0) });
 
-        const parsedCustomNoResultsPage: string = '<div><a href="&lt;span class=" coveo-highlight"="">querySearched"&gt;</a></div>';
+        const parsedCustomNoResultsPage: string = '<div><a href="<span class=" coveo-highlight"="">querySearched"&gt;</a></div>';
```

Comment:
I'm not sure how to test this. I went to the "No results" page and saw no difference.

However, I'm not sure why underscore added a `&lt;` in the first place, since opening brackets are supported in attribute values.

Reply:
Hmm, why is underscore returning the less than sign unencoded, but the greater than sign (`&gt;`) is still encoded?
Also, it seems like this was the only `<` affected in the entire code-base. Why was it only here?

Reply:
The less-than sign is part of a string while the greater-than sign is part of an innerText, which I guess is why the greater-than sign was encoded.

I don't know why this is the only `<` affected in the entire code-base.